packages/components/src/components/ExternalCard/VideoCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Icon } from '../Icon';
+import { Stack } from '../Stack';
+import {
+  StyledContent,
+  StyledDetails,
+  StyledTitle,
+  StyledWrapper,
+} from './elements';
+
+type Video = {
+  duration: string;
+  title: string;
+  thumbnail: string;
+  url: string;
+};
+
+type VideoCardProps = { onClick?: () => void } & Video;
+
+export const VideoCard: React.FC<VideoCardProps> = ({
+  duration,
+  title,
+  thumbnail,
+  url,
+  ...props
+}) => {
+  return (
+    <StyledWrapper
+      as="a"
+      direction="vertical"
+      href={url}
+      target="_blank"
+      thumbnail={thumbnail}
+      {...props}
+    >
+      <StyledContent>
+        <StyledDetails>
+          <Stack
+            align="center"
+            css={{
+              flexShrink: 0,
+              height: '32px',
+              width: '32px',
```

Comment:
In this case we can use `padding` instead if we want.

Reply:
@tristandubbeld would it be better to use the padding instead of defining the height?

Reply:
Sorry, forgot to explain why I thought this is the case. Whenever the user has a bigger personal font size preference set in the browser (and when the page properly uses `rem` values) the text might scale and be bigger than the set width and height. In this case, since we're talking about an `Icon` it doesn't matter, but when it's text it does. This is why I usually - in all cases, just to be sure and for consistency - prefer padding over a set width and height.

Reply:
@tristandubbeld oh, that makes sense. Thank you for explaining. 🙇🏽‍♀️